packages/protocol/contracts/governance/test/MockValidators.sol

```diff
@@ -23,6 +23,14 @@ contract MockValidators is IValidators {
     return validators;
   }
 
+  function numberValidatorsInCurrentSet() external view returns (uint256) {
```

Comment:
Why do we need this in `MockValidators`?

Reply:
Same as the IValidators issue below.

Reply:
Remove it?